src/main/java/org/opensrp/web/rest/PractitionerRoleResource.java

```diff
@@ -143,4 +154,18 @@ public void setPractitionerRoleService(PractitionerRoleService practitionerRoleS
         }
     }
 
+    private PractitionerRoleSearchBean createPractitionerRoleSearchBean(Integer pageNumber, Integer pageSize,
+            String orderByType, String orderByFieldName) {
+        BaseSearchBean.OrderByType orderByTypeEnum;
+        BaseSearchBean.FieldName fieldName;
+        orderByTypeEnum = orderByType != null ? BaseSearchBean.OrderByType.valueOf(orderByType) : BaseSearchBean.OrderByType.DESC;
+        fieldName = orderByFieldName != null ? BaseSearchBean.FieldName.valueOf(orderByFieldName) : BaseSearchBean.FieldName.id;
+
+        return PractitionerRoleSearchBean.builder()
+                .pageNumber(pageNumber)
+                .pageSize(pageSize)
+                .orderByType(orderByTypeEnum)
+                .orderByFieldName(fieldName).build();
+    }
```

Comment:
Can be refactored to a utils method

Reply:
As per our discussion, using the builder pattern, each populating the builder has to be done for each resource.